src/meta/server_state.cpp

```diff
@@ -56,6 +56,17 @@ using namespace dsn;
 namespace dsn {
 namespace replication {
 
+const int32_t max_allowed_replica_count = 15;
```

Comment:
Why the `max` is not set `dynamic variable` like `min`?

Reply:
Initially the `max_allowed_replica_count ` was set configurable. Then as discussed with @hycdong above, `max_allowed_replica_count ` has been changed as a const value, not a configurable one, since in normal cases it will not be updated.

Reply:
By the way, the const int is named like `k...`  refer to [google code style](https://google.github.io/styleguide/cppguide.html#Constant_Names)

Reply:
Agreed. I'd considered following google code style such as the camel-case naming of constant which begins with 'k'. On the other hand, however I'd found many constants are named in upper or lower snake case before I turned to follow the existing custom. I think we should gradually unify the naming style.